### PR TITLE
Adds Anthromorphic Maniraptorans

### PR DESCRIPTION
**Pull request by lpeapnni**
## About The Pull Request
adds Anthromorphic Maniraptorans to the game
image

compatible with most clothing!
image
image

## Why It's Good For The Game
more species are always good, and i know of multiple people who would be appreciative of Anthromorphic Maniraptorans being added to fallout 13

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
add: Added Anthromorphic Maniraptorans
/:cl:

### CODE REVIEW

**Comment by hubertjones** [comment]
Nice try.
image

**Comment by lpeapnni** [comment]
> Nice try. image

image
image

**Comment by mopcat** [comment]
image

**Comment by mazzinsanity** [comment]
Still ugly but I guess it's fine as long as no-one makes actual Vox characters

**Comment by hubertjones** [comment]
> > Nice try. image
> 
> image image

image

**Comment by Stutternov** [comment]
to highlight last one was closed due to community cord telling me to close it and code staff doing so as well.

(I do not care i will make a shitbird either way someday)

**Comment by electrical-pants** [comment]
if people are insistent then there should be vox markings and stuff (like introduced with the customization PR, but without the actual vox namings)

i think there's a fine line people ride between having suspension of disbelief around furry characters and then just seeing Furry Space Aliens in the form of vox, teshari, etc. renaming them to whatever is essentially redundant because i'm certain as long as the actual species exists, people will just play them like vox regardless of their name or implied difference

which creates potential administration issues and other stuff i don't really want to see and i don't want the staff to have to deal with

TLDR : the customization options with the specific vox names removed are enough, and just putting the vox species in with a basic rename is practically giving people tacit authorization to make vox characters, which is bad

if you wanna play a freeky anthro play something that at least tries to fit with the setting, base furries require enough suspension of disbelief as is and so adding this is Bad IMHO

**Comment by maaacha** [comment]
Not wanted.

**Comment by lpeapnni** [comment]
> Not wanted.

not wanted by a vocal minority but whatever
